lib/version.js

```diff
@@ -1,4 +1,8 @@
+const supportedVersions = ['1.8', '1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15', '1.16', '1.17', '1.18', '1.19', '1.20']
+const testedVersions = ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.5', '1.17.1', '1.18.2', '1.19', '1.19.2', '1.19.3', '1.19.4', '1.20.1']
 module.exports = {
-  supportedVersions: ['1.8', '1.9', '1.10', '1.11', '1.12', '1.13', '1.14', '1.15', '1.16', '1.17', '1.18', '1.19', '1.20'],
-  testedVersions: ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.5', '1.17.1', '1.18.2', '1.19', '1.19.2', '1.19.3', '1.19.4', '1.20.1']
+  supportedVersions,
+  testedVersions,
+  latestSupportedVersion: testedVersions[testedVersions.length - 1],
+  oldestSupportedVersion: testedVersions[0]
```

Comment:
Added these two variables to the versions.js. So if there is a change and tested versions is not the latest supported, it can be set here. Otherwise if needed we can also manually define these instead of being first/last tested versions.

Reply:
that makes sense yeah; but I think we'll keep testing supported versions

Reply:
in fact I suggest we expose these 2 variables in https://github.com/PrismarineJS/mineflayer/blob/master/lib/loader.js#L58 (there and in the doc) and stop exposing supportedVersions